PublicAPI/Microsoft.ApplicationInsights.dll/net46/PublicAPI.Unshipped.txt

```diff
@@ -53,4 +53,7 @@ Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrappe
 Microsoft.ApplicationInsights.Extensibility.SequencePropertyInitializer.SequencePropertyInitializer() -> void
 Microsoft.ApplicationInsights.Extensibility.Implementation.OperationTelemetry.OperationTelemetry() -> void
 Microsoft.ApplicationInsights.Extensibility.OperationCorrelationTelemetryInitializer.OperationCorrelationTelemetryInitializer() -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.ApplicationId.DictionaryApplicationIdProvider.DictionaryApplicationIdProvider() -> void
\ No newline at end of file
+Microsoft.ApplicationInsights.Extensibility.Implementation.ApplicationId.DictionaryApplicationIdProvider.DictionaryApplicationIdProvider() -> void
```

Comment:
why do they have to be public APIs? If they are needed in ServerTelemteryChannel, could they be defined there instead?

Reply:
I talked to Dmitry offline about this. I'm not happy adding extension methods to the public api and I'm open to alternatives.

We have two channels and they're in different projects. Both channels need access to these extension methods.

Reply:
copy-paste is one option. shared file compiled into both project is another one.